package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Frontend for the supporter platform.",
   "scripts": {
     "clean": "echo 'Cleaning Public Folder' &&  rimraf public/*",
-    "lint": "echo 'Validating JS' && eslint --ext .js --ext .jsx .",
+    "validate": "echo 'Validating JS' && npm-run-all lint flow",
```

Comment:
By default `npm-run-all` runs the given tasks in serial mode. In order to make it readable, it would be nice to add the `--serial` argument.